Adds documentation for mautic/mautic/pull/14100 (#317)
mautic/user-documentation: docs/components/forms.rst

   .. note:: 
      You can associate checkbox group fields with *boolean* and *select - multiple* fields, but not *select* fields.

>    - **Date** - This field allows the visitor to select a date with a calendar picker. The formatting of the date applies the default setting in your Configuration. [A PHP parseable BNF-like string](https://www.php.net/manual/en/datetime.formats.php) can be used to set a dynamic/relative to now default value for this field.

Reviewer: Hi @putzwasser please use the command `make link` at the terminal, follow the steps and then use the xref macro here. For info, if you're doing internal linking check the docs on using the [:ref: role](https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html#the-ref-role) for linking within the same document, and using [:doc: to link across pages](https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html#the-doc-role).

Please fix this in all the places you've mentioned it. Thanks!

>    - **Date** - This field allows the visitor to select a date with a calendar picker. The formatting of the date applies the default setting in your Configuration. [A PHP parseable BNF-like string](https://www.php.net/manual/en/datetime.formats.php) can be used to set a dynamic/relative to now default value for this field.

Reviewer: ```suggestion
   - **Date** - This field allows the visitor to select a date with a calendar picker. The formatting of the date applies the default setting in your Configuration. You can use [A PHP parseable BNF-like string](https://www.php.net/manual/en/datetime.formats.php) to set a dynamic/relative to now default value for this field.
```

Passive -> Active voice
